CONTRIBUTING_java.md

```diff
@@ -202,6 +202,8 @@ from the root of the project, and then view the coverage report in the `modules/
 open modules/coverage-reports/target/site/jacoco-aggregate/index.html
 ```
 
+To obtain an coverage report of only integration tests, run `mvn clean verify -Dskip.surefire.tests`.
```

Comment:
If you follow through with the name change in the java template PR, then this needs to be changed also:
```suggestion
To obtain an coverage report of only integration tests, run `mvn clean verify -Dskip.unit.tests`.
```